posthog/models/person/person.py

```diff
@@ -32,6 +32,14 @@ def distinct_ids(self) -> List[str]:
             .values_list("distinct_id")
         ]
 
+    def get_distinct_ids_with_version(self) -> Dict[str, int]:
```

Comment:
Let's not add _anything_ to models if we can avoid it - people will find it, abuse it and it's hard to remove. Let's inline this `_delete_ch_distinct_ids`